plone/app/contenttypes/setuphandlers.py

```diff
@@ -307,7 +307,8 @@ def configure_members_folder(portal, target_language):
         container = addContentToContainer(portal, container)
         container.setOrdering('unordered')
         container.reindexObject()
-        _publish(container)
+        
+        
```

Comment:
too many blank lines. please read and follow: https://docs.plone.org/develop/styleguide/index.html including how to set up your editor.